packages/rich-text/src/helpers/sanitizeIncomingSlateDoc.ts

```diff
@@ -0,0 +1,25 @@
+import { CustomText, TextOrCustomElement } from '../types';
+
+const isTextElement = (node: TextOrCustomElement): node is CustomText => 'text' in node;
+
+/**
+ * Ensures all nodes have a child leaf text element. This should be handled by
+ * Slate but its behavior has proven to be buggy and unpredictable.
```

Comment:
Is there any Slate issue we could link to?

Reply:
I couldn't find any